aleph/logic/audit.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from aleph.core import db, celery
+from aleph.model import Audit
+
+log = logging.getLogger(__name__)
+
+ACTIVITY_SCHEMA = {
+    "USER.LOGIN": [],
+    "USER.SEARCH": ["query_string", "filters"],
+    "USER.VIEW_COLLECTION": ["collection_id", ],
+    "USER.VIEW_DOCUMENT": ["document_id", ]
+}
+
+
+@celery.task(priority=1)
+def record_audit(activity_type, activity_data, authz):
```

Comment:
I'm pretty convinced that we cannot put an `authz` object on the queue (it'd need to be JSON serialisable). So I think we need a record_audit function (which takes the authz) and which in turn calls record_audit_delayed (which would be on the queue).